tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+
+
+1. Click `Save Changes`
+
+After creating an Application with your OIDC provider, you will get the following credentials for setting up RBAC for your remote cluster:
+
+* Domain - to be used as `issuerURL` in the sections below
+* Client ID - to be used as `CLIENT-ID` in the sections below
+* Client Secret - to be used as `CLIENT-SECRET` in the sections below
+
+For more details, please refer to [Auth0 Setup Walkthrough](https://backstage.io/docs/auth/auth0/provider) in the Backstage documentation. To configure other OIDC providers, please refer to [Authentication in Backstage](https://backstage.io/docs/auth/).
+
+## <a id="configure-cluster"></a> Configure the Kubernetes cluster with the OIDC provider
+
+These steps will help you to configure the cluster with the OIDC provider's credentials.
+
+1. Create a file with the contents below and name it `rbac-setup.yaml`. The following example applies to EKS clusters.
+
+```console
+apiVersion: eksctl.10/vialphas
+kind: ClusterConfig
+metadata:
+    name: "CLUSTER-NAME"
+    region: "AWS-REGION"
+identitvProviders:
+  - name: autho
+    type: oidc
+    issuerUrl: "ISSUER-URL"
+    clientId: "CLIENT-ID"
+    usernameClaim: email
+```
+Where:
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `AWS-REGION` is the AWS region of the EKS cluster
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+
+
+2. Using `eksctl`, execute the following command:
+
+```console
+eksctl associate identityprovider -f rbac-setup.yaml
+```
+
+3. To check if the association of the OIDC provider with the EKS cluster was successful, run the following command
+
+```console
+eksctl get identityprovider --cluster `CLUSTER-NAME`
+```
+
+The output should state `ACTIVE` in the STATUS column.
+
+
+## <a id="configure-tap-gui"></a> Configure the Tanzu Application Platform GUI
+
+The next step is to configure visibility of the remote cluster in Tanzu Application Platform GUI. 
+
+1. Make sure you added an `auth` section to the `app_config` file taht Tanzu Application Platform GUI uses. In the example for Auth0, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+auth:
+    environment: development
+    providers:
+        autho:
+            development:
+                clientId: "CLIENT-ID"
+                clientSecret: "CLIENT-SECRET"
+                domain: "ISSUER-URL"
+```
+Where:
+
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-SECRET` is the Client Secret you obtained while setting up the OIDC provider
+
+
+2. You must also add a `kubernetes` section to the `app_config` file that Tanzu Application Platform GUI uses. This section must have an entry for each cluster that has resources to view.
```

Comment:
Maybe we could say "`app_config` stanza" instead of "`app_config` file" here? It kinda sounds like we're expecting users to maintain a separate file for their TAP GUI app config, and I don't think that's what we want.

Reply:
Fixed, thanks